larky-api/src/main/java/com/verygood/security/larkyapi/LarkyAPIEngine.java

```diff
@@ -0,0 +1,39 @@
+package com.verygood.security.larkyapi;
+
+import com.verygood.security.larky.jsr223.LarkyScriptEngine;
+import com.verygood.security.larky.parser.ParsedStarFile;
+
+import javax.script.ScriptContext;
+import javax.script.ScriptException;
+import java.io.Reader;
+
+public class LarkyAPIEngine extends LarkyScriptEngine {
+
+    private Object extractStarfileOutput(ParsedStarFile starfile, String outputVar) {
+        return starfile.getGlobalEnvironmentVariable(outputVar, Object.class);
+    }
+
+    public Object executeScript(String script, String outputVar)
+        throws ScriptException, NullPointerException {
```

Comment:
Also, `NullPointerException` is a runtime error, why would we put it as part of the method parameter name?